src/constants/specs.js

```diff
@@ -1287,19 +1301,6 @@ export const specs = {
         },
         user_props: ["handleChange"]
       },
-      {
```

Comment:
Why did you delete this?

Reply:
Because `paper` model doesn't have a `contribution` field.